MAINTAINERS.md

```diff
@@ -1,5 +1,55 @@
 # Maintaining lorri
 
+## Versioning scheme
+
+The versioning scheme for lorri is `MAJOR.MINOR`, where `MAJOR` and `MINOR` are
+non-negative integers without leading zeroes.
```

Comment:
```suggestion
non-negative integers without leading zeroes. We follow the same rules as semver,
and remainder of this section describes exactly what that means for lorri.
```

Reply:
I mentioned semver in a previous iteration. Then I took it out, because I think that it doesn't really help.
- semver explicitly uses `MAJOR.MINOR.PATCH` whereas this document proposes `MAJOR.MINOR`
- semver talks about "API backwards compatibility", which only applies in an abstract sense to a command line tool

For those reasons, I came to the conclusion that "it's like semver, but ..." is less clear than just explaining the scheme directly. It's definitely semver-inspired, but I think of semver as a complete versioning scheme that one should adopt or not adopt. "Like semver, but ..." just doesn't really aid understanding in my opinion.

Reply:
Seems fine enough to me. We're not in the library business or production operations business. Maybe one day this will come back to bite us, but if it does, we can do something different then.